src/services/tests/hubApi.test.ts

```diff
@@ -0,0 +1,15 @@
+import { queryParamsSerializer } from "../hubApi";
+describe("parameter serializer", () => {
+  it("serializes params", () => {
+    const paramsObject = {
+      kind: ["encoder"],
+      type: ["script"],
+      somethingUndefined: undefined,
+      emptyArray: [],
+    };
+
+    expect(queryParamsSerializer(paramsObject)).toEqual(
```

Comment:
The name is confusing, because as a noun it implies that it is an object. Our functions usually are verbs. So maybe change it to serializeQueryParams ?